core/src/test/java/apoc/export/graphml/ExportGraphMLTest.java

```diff
@@ -182,6 +187,22 @@ public void testImportGraphML() throws Exception {
         TestUtil.testCall(db, "MATCH  (c:Bar {age: 12, values: [1,2,3]}) RETURN COUNT(c) AS c", null, (r) -> assertEquals(1L, r.get("c")));
     }
 
+    @Test
+    public void testImportGraphMLLargeFile() {
```

Comment:
@vga91 does that test fail without your fix?

Reply:
@fbiville Exactly. Throws an `org.neo4j.kernel.impl.util.collection.MemoryAllocationLimitException` like the related issues.